src/getAppSyncConfig.js

```diff
@@ -78,6 +78,7 @@ export default function getAppSyncConfig(context, appSyncConfig) {
             environment: {
               ...context.serverless.service.provider.environment,
               ...func.environment,
+              ...(context.options.lambda.loadLocalEnv === true ? process.env : {}),
```

Comment:
I did this on purpose because in our use case if the same variable is defined in environment (`.env`) and in `serverless.yml`, we want the variable from the envirnoment to be used (to have higher priority).

But okay, I get your point. I will refactor it as you suggest and I will create new PR where I will add option "importValueMap" because we use a lot of `Fn::ImportValue` and this should do the job for us. 👍 
Then we will be able to do:
```yaml
appsync-simulator:
    importValueMap:
      some-import-name: ${env:SOME_IMPORT, default_value}
```